repos/system_upgrade/el7toel8/actors/repositoriesmapping/tests/unit_test_repositoriesmapping.py

```diff
@@ -109,22 +109,6 @@ def test_scan_valid_file_with_comments(monkeypatch, arch, src_type, dst_type):
     assert api.produce.model_instances == [gen_RepositoriesMap(expected_records)]
 
 
-@pytest.mark.parametrize('isFile,err_summary', [
-    (False, 'The repository mapping file not found'),
-    (True, 'The repository mapping file is invalid'),
-])
-def test_scan_missing_or_empty_file(monkeypatch, isFile, err_summary):
-    monkeypatch.setattr(api, 'current_actor', CurrentActorMocked(architecture.ARCH_X86_64))
-    monkeypatch.setattr(api, 'produce', produce_mocked())
-    monkeypatch.setattr('os.path.isfile', lambda dummy: isFile)
-    if isFile:
-        monkeypatch.setattr('os.path.getsize', lambda dummy: 0)
-    with pytest.raises(StopActorExecutionError) as err:
-        repositoriesmapping.scan_repositories()
-    assert not api.produce.called
-    assert err_summary in str(err)
-
-
```

Comment:
@drehak  That can be modified, but cannot be removed. Doesn't matter the execution error is raised inside the shared library function. The test should cover the error is in example not catched & skipped inside the function.

Reply:
Modified this test and added it back.